style.css

```diff
@@ -0,0 +1,98 @@
+@import url(https://fonts.googleapis.com/css?family=Oswald:700|Quattrocento);
+
+* {
+    box-sizing: border-box;
+}
+
+body,html {
+    min-height: 100%;
+    height: 100%;
+}
+
+body {
+    align-items: center;
+    display: flex;
+    justify-content: center;
+    margin: 0;
+    color: black;
+    font-family: Quattrocento, serif;
+    line-height: 1.8925;
+    background-image: url(img/cab-splash.png);
+    background-size: cover;
+    background-position: center;
+    &:before,
+    &:after {
```

Comment:
Forgive my potential ignorance, this smells like SASS/Less but I don't see any build tools. Is this valid CSS syntax these days??